providers/gce/gce_instances.go

```diff
@@ -551,19 +551,24 @@ func (g *Cloud) AliasRangesByProviderID(providerID string) (cidrs []string, err
 		for _, r := range networkInterface.AliasIpRanges {
 			cidrs = append(cidrs, r.IpCidrRange)
 		}
-		ipv6Addr := getIPV6AddressFromInterface(networkInterface)
-		if ipv6Addr != "" {
-			// The podCIDR range is the first /112 subrange from the /96 assigned to
-			// the node
-			ipv6PodCIDR := fmt.Sprintf("%s/112", ipv6Addr)
-			cidrs = append(cidrs, ipv6PodCIDR)
-		} else {
-			klog.Infof("No IPv6 addresses found for %s", providerID)
-		}
+		cidrs = g.AccommodateIPV6Addresses(cidrs, networkInterface, providerID)
 	}
 	return
 }
 
+func (g *Cloud) AccommodateIPV6Addresses(cidrs []string, networkInterface *compute.NetworkInterface, providerID string) []string {
```

Comment:
this function is not very intuitive, it also doesn't need to be a method , I don't know,  I will make it as `getIPv6PodCIDRFromInterface(networkInterface *compute.NetworkInterface, providerID string) string` and use it as 
````
if ipv6PodCidr := getIPv6PodCIDRFromInterface(); ipv6PodCIDR != nil {
  cidrs = append(cidrs, ipv6PodCIDR
}
```

whatever you choose, add a comment describing it